wavelink/player.py

```diff
@@ -336,7 +336,7 @@ async def connect(self, *, timeout: float, reconnect: bool, **kwargs: Any) -> No
 
         await self.channel.guild.change_voice_state(channel=self.channel, **kwargs)
 
-    async def move_to(self, channel: discord.VoiceChannel) -> None:
+    async def move_to(self, channel: discord.VoiceChannel, self_mute: bool = False, self_deaf: bool = False) -> None:
```

Comment:
Could you please change the new areguments to keyword arguments.
E.g `..., *, self_mute=, self_deaf=`